### DISCUSSION TITLE
I want to use SLIT to develop commercial applications. Do I need to pay?

### DISCUSSION THREAD

**Discussion by pcevent** [slint-ui/slint#2972] [Q&A] [answered]
I want to use SLIT to develop commercial applications. Do I need to pay?

**Comment by ogoffart** [comment, marked as answer]
You probably meant "Slint." 
No you don't need to pay, you can use the "royalty-free" license.
The royalty free allow you to create desktop application for free under any license, as long as you mention that you use Slint in the About dialog. 

https://github.com/slint-ui/slint/blob/master/LICENSES/LicenseRef-Slint-Royalty-free-1.0.md

https://github.com/slint-ui/slint/blob/master/FAQ.md#licensing

**Comment by pcevent** [comment]
Okay, thank you!
